scripts/closure-compiler/tsconfig-rxjs.json

```diff
@@ -0,0 +1,16 @@
+{
```

Comment:
maybe extend from the tsconfig.json in rxjs sources? Shouldn't you share most of their options?